src/Lykke.Job.BlockchainCashoutProcessor/Settings/Blockchain/BlockchainSettings.cs

```diff
@@ -1,14 +1,23 @@
 ﻿using JetBrains.Annotations;
+using Lykke.SettingsReader.Attributes;
 
 namespace Lykke.Job.BlockchainCashoutProcessor.Settings.Blockchain
 {
     [UsedImplicitly]
     public class BlockchainSettings
```

Comment:
Btw, setting url for this Job contains section Blockchain**s** but not Blockchain, and contains some more fields than this class does (while not containing the `IsDisabled` prop). Will it match in runtime? Or do I misunderstand the logic?

Reply:
```BlockchainSettings``` is item model of the ```BlockchainCashoutProcessorSettings```. ```IsDisabled``` is optional field, which is just ommited for every items on dev/test